Niyonizeye/AnnounceIT
# login page is updated where user can signin for purpose of viewing an…

### What does this PR do
- User Should Signin for purpose of view The announcement created
### description of task to be completed
- Added the Html
- Added Style
### how should this be manually tested
- Clone git repository
- https://github.com/Niyonizeye/AnnounceIT.git
- open index.html in the browser
- See on Top Right of Page
- Click on Login
### Any background context you want to provide?
- N/A

What are relevant pivotal tracker stories ?
- https://www.pivotaltracker.com/story/show/170770778

### screenshots (if appropriate)
![login](https://user-images.githubusercontent.com/45454232/73019381-25b01000-3e2c-11ea-90d3-122ef80827dd.PNG)


### Questions
N/A
